docs/content/en/routing.md

```diff
@@ -106,6 +106,12 @@ If on `Nuxt` version lower than 2.10.2, and using strategy `prefix_except_defaul
 
 In some cases, you might want to translate URLs in addition to having them prefixed with the locale code. There are 2 ways of configuring custom paths for your pages: in-component options or via the module's configuration.
 
+<alert type="warning">
+
+Custom paths are not supported with the `no-prefix` [strategy](#strategy).
```

Comment:
I think that linking to `strategy` might not be necessary. But I guess it's no big deal.